Sources/QuickObjectiveC/QuickSpec.m

```diff
@@ -109,22 +111,24 @@ + (void)buildExamplesIfNeeded {
  @return The selector of the newly defined instance method.
  */
 + (SEL)addInstanceMethodForExample:(Example *)example classSelectorNames:(NSMutableSet<NSString*> *)selectorNames {
-    IMP implementation = imp_implementationWithBlock(^(QuickSpec *self){
+    IMP implementation = imp_implementationWithBlock(^(QuickSpec *self, void (^completionHandler)(void)){
         self.example = example;
         currentSpec = self;
-        [example run];
+        [example runWithCompletionHandler:completionHandler];
     });
 
-    const char *types = [[NSString stringWithFormat:@"%s%s%s", @encode(void), @encode(id), @encode(SEL)] UTF8String];
+    const char *types = [[NSString stringWithFormat:@"%s%s%s@?<v@?>", @encode(void), @encode(id), @encode(SEL)] UTF8String];
```

Comment:
This was frustrating. Initially, I had this use whatever came from `@encode(void (^)(void))`, but that was apparently producing incorrect typing information. After investigating what the invocation is for an XCTestCase async test, I copied that type information and used it here. This caused XCTest to correctly pass in the necessary completion handler.